# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../lprof/bool.patch
# Copyright (C) 2010 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- lprof-1.11.4.1/src/liblprof/lcmsprf.h_orig	2010-08-10 21:46:30.000000000 +0000
+++ lprof-1.11.4.1/src/liblprof/lcmsprf.h	2010-08-10 21:47:22.000000000 +0000
@@ -67,6 +67,7 @@
 #define mmax(a,b) ((a) > (b)?(a):(b))
 #endif
 
+#define BOOL int
 
 /* Misc operations ------------------------------------------------------------------------ */
 
